services/core-web/src/components/mine/Tailings/MineTailingsTable.js

```diff
@@ -38,6 +37,7 @@ const defaultProps = {};
 const MineTailingsTable = (props) => {
   const {
     TSFOperatingStatusCodeHash,
+    // eslint-disable-next-line no-unused-vars
     consequenceClassificationStatusCodeHash,
```

Comment:
Is this being used or not? Wondering why the eslint line was necessary.

Reply:
Updated the PR to remove these for this page.

You'll find a lot more of these in the system than I'd like, but we can work on clearing them out as we go.

There are some interactions with Redux that seem to cause some props to be flagged as unused when they are not and I think often this is why those flags were set.